src/Resource.js

```diff
@@ -308,15 +308,26 @@ Resource.prototype._loadImage = function () {
  * @private
  */
 Resource.prototype._loadElement = function (type) {
-    this.data = document.createElement(type);
+    if (type === 'audio' && typeof Audio !== 'undefined') {
```

Comment:
I can test it for video and review this pull request, note that Webview+ and other runtimes work for audio/video without these changes.